packages/agoric-cli/src/bin-agops.js

```diff
@@ -30,22 +31,22 @@ program.addCommand(await makeOracleCommand(logger));
 program.addCommand(await makeEconomicCommiteeCommand(logger));
 program.addCommand(await makePerfCommand(logger));
 program.addCommand(await makePsmCommand(logger));
-program.addCommand(await makeReserveCommand(logger));
 program.addCommand(await makeVaultsCommand(logger));
 
-program.addCommand(
-  await makeInterCommand(
-    {
-      env: { ...process.env },
-      stdout: process.stdout,
-      stderr: process.stderr,
-      createCommand,
-      execFileSync,
-      now: () => Date.now(),
-    },
-    { fetch },
-  ),
-);
+const procIO = {
+  env: { ...process.env },
+  stdout: process.stdout,
+  stderr: process.stderr,
+  createCommand,
+  execFileSync,
+  now: () => Date.now(),
+  clock: () => Promise.resolve(Date.now()),
```

Comment:
this is redundant with `now()`. the one use of I can find clock,
```js
const t0 = await clock();
```
should be be,
```js
const t0 = now();
```

Reply:
`now()` is synchronous I/O, which is a pointy instrument, and not strictly needed, so I didn't want to grant it. But now this is inconsistent. sigh. yeah, I suppose I can change it back. oh... did I put both `now` and `clock` in the same `io` object? ew.

Reply:
> What's the harm in passing `Date.now()`?

It's synchronous I/O, which, as I say, is a pointy instrument. The ability to read the clock twice in the same turn is a very powerful thing that only stuff below the platform needs. (weird... I can't find where we discussed synchronous I/O vs. "all I/O happens between turns" before, but I remember it vividly)

> all the code in the CLI can get it from global

for now, yes. I'm making slow progress on fixing that, though: https://github.com/LavaMoat/LavaMoat/issues/228#issuecomment-1486624429